---
title: Systematic analysis of alternative first exons in plant genomes
publication_types:
  - "2"
authors:
  - Weihua-Chen 
  - Guanting Lv
  - Congying Lv
  - Changqing Zeng
  - Songnian Hu
publication_short: ""
abstract: >-
  Background


  Alternative splicing (AS) contributes significantly to protein diversity, by selectively using different combinations of exons of the same gene under certain circumstances. One particular type of AS is the use of alternative first exons (AFEs), which can have consequences far beyond the fine-tuning of protein functions. For example, AFEs may change the N-termini of proteins and thereby direct them to different cellular compartments. When alternative first exons are distant, they are usually associated with alternative promoters, thereby conferring an extra level of gene expression regulation. However, only few studies have examined the patterns of AFEs, and these analyses were mainly focused on mammalian genomes. Recent studies have shown that AFEs exist in the rice genome, and are regulated in a tissue-specific manner. Our current understanding of AFEs in plants is still limited, including important issues such as their regulation, contribution to protein diversity, and evolutionary conservation.


  Results


  We systematically identified 1,378 and 645 AFE-containing clusters in rice and Arabidopsis, respectively. From our data sets, we identified two types of AFEs according to their genomic organisation. In genes with type I AFEs, the first exons are mutually exclusive, while most of the downstream exons are shared among alternative transcripts. Conversely, in genes with type II AFEs, the first exon of one gene structure is an internal exon of an alternative gene structure. The functionality analysis indicated about half and ~19% of the AFEs in Arabidopsis and rice could alter N-terminal protein sequences, and ~5% of the functional alteration in type II AFEs involved protein domain addition/deletion in both genomes. Expression analysis indicated that 20~66% of rice AFE clusters were tissue- and/or development- specifically transcribed, which is consistent with previous observations; however, a much smaller percentage of Arabidopsis AFEs was regulated in this manner, which suggests different regulation mechanisms of AFEs between rice and Arabidopsis. Statistical analysis of some features of AFE clusters, such as splice-site strength and secondary structure formation further revealed differences between these two species. Orthologous search of AFE-containing gene pairs detected only 19 gene pairs conserved between rice and Arabidopsis, accounting only for a few percent of AFE-containing clusters.


  Conclusion


  Our analysis of AFE-containing genes in rice and Arabidopsis indicates that AFEs have multiple functions, from regulating gene expression to generating protein diversity. Comparisons of AFE clusters revealed different features in the two plant species, which indicates that AFEs may have evolved independently after the separation of rice (a model monocot) and Arabidopsis (a model dicot).

draft: false
featured: false
tags:
  - BMC plant biology
slides: null
url_pdf: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2174465/pdf/1471-2229-7-55.pdf
image:
  caption: ""
  focal_point: ""
  preview_only: false
summary: ""
url_dataset: ""
url_project: ""
url_source: ""
url_video: ""
author_notes: []
doi: 10.1186/1471-2229-7-55
publication: BMC plant biology
projects: []
date: 2007-10-15T00:00:00.000Z
url_slides: ""
publishDate: 2017-01-01T00:00:00.000Z
url_poster: ""
url_code: ""
---

{{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

