---
title: "ColorTree: a batch customization tool for phylogenic trees"
publication_types: ["2"]
# Author notes (optional)
authors: 
  - Weihua-Chen
  - Martin J Lercher
# Author notes (optional)
author_notes: []

publication_short: 
abstract: >-
  Background


  Genome sequencing projects and comparative genomics studies typically aim to trace the evolutionary history of large gene sets, often requiring human inspection of hundreds of phylogenetic trees. If trees are checked for compatibility with an explicit null hypothesis (e.g., the monophyly of certain groups), this daunting task is greatly facilitated by an appropriate coloring scheme.


  Findings


  In this note, we introduce ColorTree, a simple yet powerful batch customization tool for phylogenic trees. Based on pattern matching rules, ColorTree applies a set of customizations to an input tree file, e.g., coloring labels or branches. The customized trees are saved to an output file, which can then be viewed and further edited by Dendroscope (a freely available tree viewer). ColorTree runs on any Perl installation as a stand-alone command line tool, and its application can thus be easily automated. This way, hundreds of phylogenic trees can be customized for easy visual inspection in a matter of minutes.


  Conclusion


  ColorTree allows efficient and flexible visual customization of large tree sets through the application of a user-supplied configuration file to multiple tree files.
draft: false
featured: ture
tags:
  - BMC research notes
slides: null
url_pdf: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2727521/pdf/1756-0500-2-155.pdf
image:
  caption: ""
  focal_point: ""
  preview_only: false
summary: ""
url_dataset: ""
url_project: ""
url_source: ""
url_video: ""

doi: 10.1186/1756-0500-2-155
publication: BMC research notes
projects: []
date: 2009-07-31T00:00:00.000Z
url_slides: ""
publishDate: 2017-01-01T00:00:00.000Z
url_poster: ""
url_code: ""
---

